src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -1,27 +1,54 @@
 (ns status-im2.contexts.wallet.account.tabs.view
   (:require
+    [clojure.string :as string]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.account.tabs.about.view :as about]
     [status-im2.contexts.wallet.account.tabs.dapps.view :as dapps]
     [status-im2.contexts.wallet.common.activity-tab.view :as activity]
     [status-im2.contexts.wallet.common.collectibles-tab.view :as collectibles]
     [status-im2.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [status-im2.contexts.wallet.common.temp :as temp]
     [status-im2.contexts.wallet.common.token-value.view :as token-value]
-    [utils.i18n :as i18n]))
+    [status-im2.contexts.wallet.common.utils :as utils]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn prepare-token
+  [{:keys [symbol marketValuesPerCurrency] :as item} color]
+  (let [fiat-value                      (utils/total-per-token item)
```

Comment:
how we are calculating `fiat-value` should be its own sub imo.
cc @ulisesmac

Reply:
Right. We can attach the token data along with the balances on each account. So that, the UI don't need to have two subs.

Reply:
Yes! @smohamedjavid @J-Son89 

`parse-tokens` could be processed by re-frame in a sub, that sub would depend on `:wallet/tokens`